Dockerfile

```diff
@@ -37,24 +38,33 @@ RUN echo "deb http://ftp.debian.org/debian stretch-backports main" >> /etc/apt/s
         echo 'opcache.revalidate_freq=2'; \
         echo 'opcache.fast_shutdown=1'; \
         echo 'opcache.enable_cli=1'; \
-    } > /usr/local/etc/php/conf.d/opcache-recommended.ini
-
-# Install wp-cli, configure apache, add scripts, create install directory & symlink
-RUN curl -s \
-        -o /usr/local/bin/wp https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar \
-        -o /run.sh https://raw.githubusercontent.com/visiblevc/wordpress-starter/master/run.sh \
-    && chmod +x /usr/local/bin/wp /run.sh \
-    && curl -s \
-        https://raw.githubusercontent.com/wp-cli/wp-cli/master/utils/wp-completion.bash | \
-        sed -e "s/wp cli completions/wp cli completions --allow-root/" > /etc/bash_completion.d/wp-cli \
+    } > /usr/local/etc/php/conf.d/opcache-recommended.ini \
     && sed -i 's/AllowOverride None/AllowOverride All/g' /etc/apache2/apache2.conf \
-    && echo "ServerName localhost" | tee /etc/apache2/conf-available/fqdn.conf && a2enconf fqdn \
+    # Fixes issue where error is logged stating apache could not resolve the
+    # fully qualified domain name
+    && echo 'ServerName localhost' > /etc/apache2/conf-available/fqdn.conf \
+    # Grab and install wp-cli from remote
+    && curl \
+        -o /usr/local/bin/wp https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar \
+        -o /etc/bash_completion.d/wp-cli https://raw.githubusercontent.com/wp-cli/wp-cli/master/utils/wp-completion.bash \
+    && a2enconf fqdn \
     && a2enmod rewrite expires \
-    && service apache2 restart \
-    && mkdir -p /app ~/.wp-cli \
+    && service apache2 restart
+
+# Install wp-cli, add scripts, create install directory & symlink
```

Comment:
This doesn't install wp-cli anymore, it's done above

Reply:
Good catch.